app/edge/views/edge-stacks/createEdgeStackView/create-edge-stack-view.controller.js

```diff
@@ -182,8 +187,9 @@ export default class CreateEdgeStackViewController {
     const templateId = this.$state.params.templateId;
     if (templateId) {
       this.preSelectTemplate(templateId);
+    } else {
+      this.state.loading = false;
     }
```

Comment:
```ts
 if (templateId) {
    await this.preSelectTemplate(templateId);
  }
this.state.loading = false;
```

since we're awaiting setTemplatesValues, `preSelectTemplate` will only return after it's finished. so adding await here will make sure we set loading to false only after everything is done